auth/token.go

```diff
@@ -6,20 +6,21 @@ package auth
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"time"
+
 	"github.com/lestrrat-go/jwx/jwt"
 	"github.com/lestrrat-go/jwx/jwt/openid"
-	"log"
-	"time"
 )
 
 const (
-	givenName       = "given_name"
-	familyName      = "family_name"
-	email           = "email"
-	sapGlobalUserID = "user_uuid"
-	sapGlobalZoneID = "zone_uuid" // tenant GUID
-	iasIssuer       = "ias_iss"
+	claimGivenName       = "given_name"
+	claimFamilyName      = "family_name"
+	claimEmail           = "claimEmail"
```

Comment:
i guess i must be claimEmail = "email"

Reply:
Good catch :-) Looks like I missed this refactoring automatism